loader/src/main/scala/com/snowplowanalytics/snowplow/storage/bigquery/loader/metrics.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2018-2020 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.snowplow.storage.bigquery
+package loader
+
+import java.util.concurrent.TimeUnit
+import org.slf4j.LoggerFactory
+import com.codahale.metrics.{Gauge, MetricRegistry, Slf4jReporter}
+
+import scala.util.control.Exception
+
+object metrics {
```

Comment:
Lowecase object names are bit unusual.

Reply:
I can see what @dilyand is trying to achieve here and I agree with the notation.
`metrics` feels like a bag related functions that otherwise would have to be pulled as
`import Metrics._` which suggests a logical construct whereas `import metrics._` suggests a package grab-bag.

Reply:
Makes sense. Maybe we need to apply this wider.

Reply:
@peel explained it better than I would have. I think I've seen it in several places already and thought we were applying it more widely 🤷‍♂️ .

Reply:
In same vein:
```
object choose extends (Seq ~> Option) {
  def apply[T](s: Seq[T]) = s.headOption
}

choose(Seq(1,2,3))
choose(Seq('a','b','c'))
```
suggests operation, whereas:
```
object Choose extends (Seq ~> Option) {
  def apply[T](s: Seq[T]) = s.headOption
}

Choose(Seq(1,2,3))
Choose(Seq('a','b','c'))
```
suggests creation.